lab_09_experiments.py

```diff
@@ -131,37 +127,67 @@ def __init__(self):
         self.menu = Menu()
 
     def setup(self):
-        NUMBER_OF_WORMS = 10
+        NUMBER_OF_WORMS = 100
+        self.worms_killed = 0
+        self.in_base = True
+        self.player_speed = 300
+        self.speed_cost = 10
+        self.worm_receipt = "Not Enough Worms!!!"
+        self.show_worm_receipt = False
 
         self.scene = arcade.Scene()
 
-        self.player_sprite = Player("Mole.png", 0.25, center_x=500, center_y=100)
+        self.camera = arcade.Camera(1000, 1000)
+
+        self.base = arcade.Sprite("Base.png", 0.75, center_x=500, center_y=500)
```

Comment:
Ah. 1000 was a screen/window size, and 500 is a middle... Could these numbers be made constants?